src/pyOpenMS/pxds/KDTreeFeatureNode.pxd

```diff
@@ -5,7 +5,7 @@ cdef extern from "<OpenMS/ANALYSIS/QUANTITATION/KDTreeFeatureNode.h>" namespace
     
     cdef cppclass KDTreeFeatureNode "OpenMS::KDTreeFeatureNode":
     # wrap-doc:
-        # A node of the kD-tree with pointer to corresponding data and index
+        #  A node of the kD-tree with pointer to corresponding data and index
```

Comment:
any reason for the indentation

Reply:
Yup, the new multiline doc parser requires 2 spaces before.

Reply:
ah great! I think we should document this somewhere (e.g. accessible in our codebase)

Reply:
This was the case for years and it is properly documented in autowrap.
I really don't know why we make our lives difficult by trying to accomodate for people not reading the documentation.. Sorry for the rant